lib/plugins/features/movies/index.js

```diff
@@ -1,19 +1,32 @@
 'use strict';
 
-const Controller     = require('./controller');
-const MovieValidator = require('../../../validators/movie');
+const Controller            = require('./controller');
+const MovieCreateValidator  = require('../../../validators/movieCreate');
```

Comment:
In regards to multi-word file names, we usually opt for kebab-case as opposed to camelCase or snake_case.